bindings/wasm/examples/editor.js

```diff
@@ -299,7 +302,25 @@ require(['vs/editor/editor.main'], async function() {
       addEdit(button);
     }
   }
-  switchTo(currentName);
+  const params = new URLSearchParams(window.location.search);
+  if (window.location.hash.length > 0) {
+    const name = unescape(window.location.hash.substring(1));
+    switchTo(name);
+  } else if (params.get('url') != null) {
+    console.log(`Fetching ${params.get('url')}`);
```

Comment:
It seems weird to call a url param `url`, especially when it's not actually a URL. How about `script`?

Reply:
sure